docs/docs/animations/values.md

```diff
@@ -138,9 +143,49 @@ const Example = () => {
     </Canvas>
   );
 };
-
 ```
 
+## Selectors
+
+Sometimes we have components that renders multiple children as a function of an array. If we want to animate such a component with React Native Skia, we can use Skia Values for the array values and use these as properties for the components.
+
+To access elements in the array correpsonding to the index of the component we can use the `Selector` function in React Native Skia. The selector is used to access items in arrays (or values by keys in objects) - making it much simpler to create computed values with Skia:
+
+```tsx twoslash
+import React from "react";
+import {
+  Canvas,
+  Rect,
+  useComputedValue,
+  useLoop,
+  Selector,
+} from "@shopify/react-native-skia";
+
+const Heights = new Array(10).fill(0).map((_, i) => i * 0.1);
+
+export const Demo = () => {
+  const loop = useLoop();
+  const heights = useComputedValue(
+    () => Heights.map((_, i) => loop.current * i * 10),
+    [loop]
+  );
+
+  return (
+    <Canvas style={{ flex: 1, marginTop: 50 }}>
+      {Heights.map((_, i) => (
+        <Rect
+          key={i}
+          x={i * 20}
+          y={0}
+          width={16}
+          height={Selector(heights, (v) => v[i])}
+          color="red"
+        />
+      ))}
+    </Canvas>
+  );
+};
+```
```

Comment:
We could add a second example:
The same approach can be done using objects.

```tsx twoslash
import React from "react";
import {
  Canvas,
  Path,
  Skia,
  Selector,
} from "@shopify/react-native-skia";

const previous = Skia.Path.Make();
const current = Skia.Path.Make();

export const Demo = () => {
  const state = useValue({ previous, current });
  return (
    <Canvas style={{ flex: 1 }}>
      <Path path={Selector(state, state => state.previous)} />
      <Path path={Selector(state, state => state.current)} />
    </Canvas>
  );
};
```

Reply:
Done!